SPECS/ocaml-tapctl.spec

```diff
@@ -72,6 +72,9 @@ make install OPAM_PREFIX=%{buildroot}%{ocaml_dir} OPAM_LIBDIR=%{buildroot}%{ocam
 %{ocaml_docdir}/xapi-tapctl
 
 %changelog
+* Wed Aug 17 2022 Gael Duperrey <gduperrey@vates.fr> - 1.5.0-5.2
+- Rebuild for updated xapi from XS82ECU1011
```

Comment:
Would have been more precise, something like what there was in previous changelog entries:
```
Rebuild for ocaml-xcp-idl and rebuilt forkexecd from XS82ECU1011
```